index.bs

```diff
@@ -1162,6 +1162,12 @@ partial interface HTMLIFrameElement {
     1. Let |report-only result| be the result of calling <a abstract-op>Check
        permissions policy</a>, given |report-only policy|, |feature|, |origin|,
        and |document|'s [=Document/origin=].
+    1. Optionally, set |result| to "<code>Disabled</code>" if the user agent deems it necessary.
+
+        Note: The user agent may wish to disallow particular contexts or scripts from using the 
```

Comment:
I would dedent this by a single space, just leaving the characters aligned with the beginning of "Optionally" above. (I think [`specfmt`](https://github.com/domfarolino/specfmt) should do this for you, if you wanted to play with it.)